src/g_vslider.c

```diff
@@ -243,7 +243,7 @@ static void vslider_save(t_gobj *z, t_binbuf *b)
                 x->x_gui.x_ldx, x->x_gui.x_ldy,
                 iem_fstyletoint(&x->x_gui.x_fsf), x->x_gui.x_fontsize,
                 bflcol[0], bflcol[1], bflcol[2],
-                x->x_val, x->x_steady);
+                x->x_val/IEMGUI_ZOOM(x), x->x_steady);
```

Comment:
urgh. this looks awful.
isn't `x_val` meant to be the **real*** value that is controlled by te slider? as such, the zoom-scaling should go into the functions that interface with the GUI, not with saving/restoring.

Reply:
unfortunately not... see for example:
https://github.com/pure-data/pure-data/blob/e2ef8158e83900734d263a3fda343a47ed0c9e3a/src/g_hslider.c#L344-L348

Reply:
maybe I should have exposed the problem in the issue tracker. Just try, you'll see that init value isn't correctly saved/restored across zooming (even using copy/paste for example).

Reply:
OK I'm currently studying this simplification.

Reply:
done